### DISCUSSION TITLE
neo-tree closed by default?

### DISCUSSION THREAD

**Discussion by wulymammoth** [LazyVim/LazyVim#4125] [Q&A] [answered]
I've looked through the neo-tree docs, but can't find an/the option? Does anyone know how to configure neo-tree to be closed by default on Neovim/LazyVim start?

**Comment by dpetka2001** [comment]
Neo-tree only starts if you pass a directory as an argument to neovim. So, don't pass a directory.

**Reply by wulymammoth** [reply, marked as answer]
I want to open my directory but not neotree. I don't need it taking up space when I already know what file I need to open with Telescope.  I've figured it out: https://github.com/wulymammoth/dotfiles/blob/bfd186228b7188e8ebe8ce1f6c8239e21aa82a69/nvim/.config/nvim/lua/plugins/neotree.lua#L7
